vcf2fhir/common.py

```diff
@@ -13,14 +13,14 @@ class _Utilities(object):
 
     """
     /**
-     * generates current date into fhir format 
+     * generates current date into fhir format
      * @return date
      */
     """
     def getFhirDate():
```

Comment:
I would also like to see the naming convention being changed according to [this](https://www.python.org/dev/peps/pep-0008/#id45), here and any other place throughout the codebase.

Ideally we can also get rid of Utiltiy class and just create utility functions as discussed [here](https://stackoverflow.com/questions/19620498/how-to-create-a-utility-class-correctly). However, this suggestion is irrelevant to purpose of this PR.